### ISSUE TITLE
Add new `Layout/EmptyLinesAroundArguments` Cop

### ISSUE THREAD

**Issue by garettarrowood** [bbatsov/rubocop#5074]
While applying a fix for issue #5052 in PR #5066 , it was discovered that the autocorrection for `Style/BracesArondHashParameters` is a little aggressive. When I applied changes to stop it from removing legitimate `\n`s, [it created snippets similar to the example below](https://github.com/bbatsov/rubocop/pull/5066/files?diff=split#diff-0e7dbdd6e99e0defc2bab2cb544b42daR207).

Adding this `EmptyLine` Cop would resolve this scenario and still allow `Style/BracesArondHashParameters` autocorrection to be improved to cleaner results.  The cleaner results I refer to our found in [this example](https://github.com/bbatsov/rubocop/pull/5066/files?diff=split#diff-0e7dbdd6e99e0defc2bab2cb544b42daR683), [this example](https://github.com/bbatsov/rubocop/pull/5066/files?diff=split#diff-4ee3a03eec811b54b5ea49c57363755dR141) and [this example](https://github.com/bbatsov/rubocop/pull/5066/files?diff=split#diff-0e7dbdd6e99e0defc2bab2cb544b42daR203).

--------

## Desired behavior

For this code snippet register a `Stlye/EmptyLinesAroundArguments` offense.

```ruby
foo(

  bar: 1

)
```

Then, autocorrect that snippet to:

```ruby
foo(
  bar: 1
)
```

## Actual behavior

Above snippet registers no offense.

## Steps to reproduce the problem

See above.

## RuboCop version

```
0.51.0 (using Parser 2.4.0.0, running on ruby 2.3.3 x86_64-darwin15)
```

**Comment by Drenmi** [comment]
This cop should go in the `Layout` department, not `Style`. 🙂

**Comment by deepj** [comment]
Hm, it seems, this doesn't work properly. I have something like

```ruby
# frozen_string_literal: true

require 'csv'
require_relative '../../importer/base'

module Labels
  module Csv
    class Import < Importer::Base
```

And that issue occurs on

```
app/services/labels/csv/import.rb:4:1: C: Layout/EmptyLinesAroundArguments: Empty line detected around arguments.
require_relative '../../importer/base'
^
app/services/labels/csv/import.rb:7:1: C: Layout/EmptyLinesAroundArguments: Empty line detected around arguments.
  module Csv
```

I guess this is not expected behaviour...

**Comment by fotinakis** [comment]
We also got this cop triggered on a `require` line.

**Comment by zhisme** [comment]
I also had this cop offense in this example.


```
# bad
config.enabled = Rails.env.in?(%w[production staging])
```
` C: Layout/EmptyLinesAroundArguments: Empty line detected around arguments.`

```
# good
config.enabled = Rails.env.in?(
    %w[production staging]
    )
```
`no offense`
